scalability_and_performance/using-argocd.adoc

```diff
@@ -0,0 +1,61 @@
+[id="using-argocd"]
+= Using ArgoCD to manage cluster configuration
+include::modules/common-attributes.adoc[]
+:context: using-argocd
+
+You can use ArgoCD to implement a GitOps configuration management system with {product-title}. You can use ArgoCD to manage many {product-title} configuration files across multiple clusters.
```

Comment:
We should add a disclaimer to clarify the scope of support and explain that ArgoCD is a community software that is not supported by Red Hat. The instructions provided by Red Hat in the docs are however supported